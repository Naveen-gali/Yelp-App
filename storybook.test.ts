import path from 'path';
import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  framework: 'react-native', //ðŸ‘ˆ Manually specify the project's framework
  configPath: path.join(__dirname, '.storybook'),
  integrityOptions: {cwd: path.join(__dirname, 'src', 'assets', 'stories')},
  // Other configurations
});
