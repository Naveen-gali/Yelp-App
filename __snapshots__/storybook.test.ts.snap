// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Fonts Using Primary Fonts 1`] = `
<Text
  style={
    [
      {
        "fontSize": 30,
      },
      {
        "fontFamily": "Poppins-Bold",
      },
    ]
  }
>
  Helllo  World
</Text>
`;

exports[`Storyshots Fonts Using Secondary Fonts 1`] = `
<Text
  style={
    [
      {
        "fontSize": 30,
      },
      {
        "fontFamily": "RumRaisin-Regular",
      },
    ]
  }
>
  Helllo  World
</Text>
`;

exports[`Storyshots MyCustomIcon All Icons 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    allowFontScaling={false}
    selectable={false}
    style={
      [
        {
          "color": "#000000",
          "fontSize": 200,
        },
        undefined,
        {
          "fontFamily": "icomoon",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        {},
      ]
    }
  >
    
  </Text>
</View>
`;

exports[`Storyshots MyCustomIcon Basic 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    allowFontScaling={false}
    selectable={false}
    style={
      [
        {
          "color": undefined,
          "fontSize": 200,
        },
        undefined,
        {
          "fontFamily": "icomoon",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        {},
      ]
    }
  >
    
  </Text>
</View>
`;

exports[`Storyshots MyCustomIcon Custom Icon From Icomoon 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    allowFontScaling={false}
    selectable={false}
    style={
      [
        {
          "color": undefined,
          "fontSize": 100,
        },
        undefined,
        {
          "fontFamily": "icomoon",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        {},
      ]
    }
  >
    
  </Text>
</View>
`;

exports[`Storyshots Svgs All Svgs 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <SvgMock
    height={200}
    width={200}
  />
</View>
`;
